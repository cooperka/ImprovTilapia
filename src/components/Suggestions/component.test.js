import React from 'react';
import renderer from 'react-test-renderer';

import Component from './component';

jest.useFakeTimers();
jest.mock('../FloatingNav/component', () => 'FloatingNav');

it('renders as expected', async () => {
  const tree = renderer.create(<Component />);
  expect(tree.toJSON()).toMatchSnapshot();
});
