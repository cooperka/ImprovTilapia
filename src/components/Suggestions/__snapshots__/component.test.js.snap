// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#000",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <FloatingNav />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#000",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#0f0",
            "marginBottom": 8,
            "marginHorizontal": 16,
            "marginTop": 20,
            "textAlign": "center",
          },
          Object {
            "fontSize": 0,
          },
        ]
      }
    >
      Press a button below to get a suggestion
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "maxHeight": 140,
        "maxWidth": 500,
      }
    }
  >
    <Button
      onPress={[Function]}
      style={
        Object {
          "flex": 1,
          "marginHorizontal": 16,
        }
      }
    >
      Relationship
    </Button>
    <Button
      onPress={[Function]}
      style={
        Object {
          "flex": 1,
          "marginHorizontal": 16,
        }
      }
    >
      Occupation
    </Button>
  </View>
</View>
`;
